bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -343,10 +435,33 @@ private void createChannelsLoadpoint(int loadpointId) {
                 "Number:Length");
         createChannel(CHANNEL_LOADPOINT_VEHICLE_SOC, channelGroup, CHANNEL_TYPE_UID_LOADPOINT_VEHICLE_SOC,
                 "Number:Dimensionless");
-        createChannel(CHANNEL_LOADPOINT_VEHICLE_TITLE, channelGroup, CHANNEL_TYPE_UID_LOADPOINT_VEHICLE_TITLE,
+        createChannel(CHANNEL_LOADPOINT_VEHICLE_NAME, channelGroup, CHANNEL_TYPE_UID_LOADPOINT_VEHICLE_NAME,
                 CoreItemFactory.STRING);
 
         removeChannel(CHANNEL_LOADPOINT_HAS_VEHICLE, channelGroup);
```

Comment:
Since you are at the code anyway: Can you please remove that constant and directly insert its value here?

Reply:
`removeChannel(CHANNEL_LOADPOINT_HAS_VEHICLE, channelGroup);`

Isn't this an error? The same channel is added some lines above!?

Reply:
ok